<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectWizard</class>
 <widget class="QDialog" name="ProjectWizard">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Wizard</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="logo">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../gameplay-editor.qrc">:/res/images/logo.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>601</width>
     <height>311</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::tab-bar{alignment: left}</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideLeft</enum>
   </property>
   <widget class="QWidget" name="tabProjects">
    <attribute name="title">
     <string>Projects</string>
    </attribute>
    <widget class="QListWidget" name="listWidgetProjects">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-1</y>
       <width>611</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="frameCreateProjectTip">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-1</y>
       <width>611</width>
       <height>281</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="labelCreateProjectTip">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>341</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>23</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Create a new project</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCreateProjectTipArrow">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../gameplay-editor.qrc">:/res/images/project-new-arrow.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frameCreateProject">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>601</width>
       <height>281</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>150</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonProjectDirectory">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>100</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelProjectDirectory">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Project Location:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelProjectName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Project Name:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCreate">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>150</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Project</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditProjectName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>521</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditProjectDirectory">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>521</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <zorder>frameCreateProject</zorder>
    <zorder>listWidgetProjects</zorder>
    <zorder>frameCreateProjectTip</zorder>
   </widget>
   <widget class="QWidget" name="tabSamples">
    <attribute name="title">
     <string>Samples</string>
    </attribute>
    <widget class="QListWidget" name="listWidgetSamples">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-10</y>
       <width>611</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonNew">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>NEW</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/res/images/wizard-new.png</normaloff>:/res/images/wizard-new.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OPEN</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/res/images/wizard-open.png</normaloff>:/res/images/wizard-open.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Meta+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>tabWidget</zorder>
  <zorder>pushButtonOpen</zorder>
  <zorder>pushButtonNew</zorder>
 </widget>
 <resources>
  <include location="../gameplay-editor.qrc"/>
 </resources>
 <connections/>
</ui>
